setup.py

```diff
@@ -5,8 +5,8 @@
 
 root = os.path.abspath(os.path.dirname(__file__))
 path = lambda *p: os.path.join(root, *p)
-assert os.path.exists('README.txt'), 'Missing README.txt'
-long_desc = open(path('README.txt')).read()
+assert os.path.exists('README.md'), 'Missing README.md'
+long_desc = open(path('README.md')).read()
```

Comment:
Hmm.  The packaging process for this project involves generating a `README.txt` in restructured format from the README.md.  In theory, Pypi now supports markdown so this conversion is not needed, but it has not been tested yet.